docs/tutorials/tabular/index.md

```diff
@@ -17,6 +17,12 @@ For standard datasets that are represented as tables (stored as CSV file, parque
   Essential information about the most important settings for tabular prediction.
 :::
 
+:::{grid-item-card} How It Works
```

Comment:
Lets have this be directly after the `In-depth` tutorial in terms of order in the sidebar.